### DISCUSSION TITLE
Translation for several languages

### DISCUSSION THREAD

**Discussion by welyson1** [GDQuest/learn-gdscript#754] [Ideas]
The project is really amazing, but because it's only in two languages, maybe it's not as accessible.

I would like to contribute to the PT-BR translation. Soon you could make a procedure to submit the translation based on the existing texts. With that we would have translation quickly into all languages.

Thanks

**Comment by HumanAftrAll** [comment]
I was thinking the same thing. Maybe we can team-up on the pt-BR translation, @welyson1!

**Comment by NathanLovato** [comment]
There is already a mostly complete Portuguese translation, The problem is that we don't have the system in place to automatically integrate translations in new releases. A teammate is working on that, but this is a side project and going at a slow pace. 

Translations repository: https://github.com/GDQuest/learn-gdscript-translations/

**Reply by HumanAftrAll** [reply]
Got it. Thanks!
